library/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM fedora
+
+RUN dnf install -y java-1.8.0-openjdk.x86_64
```

Comment:
I remember reading somewhere that you should avoid using DNF or Yum in Docker images (if that doesn't make sense please ignore my comment). Can we replace the FROM with an existing image that has these components already installed?